pkg/autohealing/controller/controller.go

```diff
@@ -340,6 +344,8 @@ func (c *Controller) startMasterMonitor(wg *sync.WaitGroup) {
 		return
 	}
 
+	totalUnhealthyNodes = append(totalUnhealthyNodes, unhealthyNodes...)
```

Comment:
`append` function is not goroutine-safe.

Reply:
Good point. I will think about how to workaround it.